internal/store/update_bucket.go

```
@@ -165,7 +165,7 @@ func (s *BucketUpdateStore) GetUpdateDetails(ctx context.Context, appId string,
 		UpdateId:   update.UpdateId,
 		CreatedAt:  helpers.NormalizeTimestamp(numberUpdate).Format(time.RFC3339),
 		CommitHash: storedMetadata.CommitHash,
-		Platform:   storedMetadata.Platform,
+		Platform:   types.Platform(storedMetadata.Platform),
```

golangci-lint (2.12.2)

[error] unnecessary conversion

(unconvert)